erpnext/hr/doctype/department_approver/department_approver.py

```diff
@@ -46,4 +52,7 @@ def get_approvers(doctype, txt, searchfield, start, page_len, filters):
 				and approver.parentfield = %s
 				and approver.approver=user.name""",(d, "%" + txt + "%", parentfield), as_list=True)
 
+	if len(approvers) == 0:
+		frappe.throw(_("Please set {0} for Employee or for Department: {1}").format(field_name, employee_department))
```

Comment:
```suggestion
		frappe.throw(_("Please set {0} for the Employee or for Department: {1}")
		           .format(field_name, frappe.bold(employee_department), frappe.bold(employee.name)),
		                     title=_(field_name + " Missing"))
```
![Screenshot 2020-06-18 at 4 16 17 PM](https://user-images.githubusercontent.com/25857446/85011075-ade9d580-b17e-11ea-88c7-6a8ea65981de.png)